### ISSUE TITLE
putting this project up for adoption

### ISSUE THREAD

**Issue by drewda** [drewda/cucumber-sublime-bundle#35]
After years of using Sublime Text and writing Gerkin, I now find myself doing neither. (I'm almost nearly switched over to Visual Studio Code, and I parted ways with Cucumber on mixed terms ;)

@bubenkoff, would you like me to transfer ownership of this repo over to you and your account?

If not, I will open this invitation to another contributor to take on the project. Please note that I will be looking for a certain level of involvement before passing the project over.

**Comment by bubenkoff** [comment]
@drewda I don't think it needs a transfer but rather addition of the contributor who can help maintaining
I'm personally right now is quite busy and not really use Gerkin for now much
but @paylogic guys do, and that's the place where i used this plugin as well.
@olegpidsadnyi are you interested in taking care of this project?
